Libs/Reflex/src/main/antlr3/reflex/Reflex.g

```diff
@@ -1036,11 +1036,18 @@ QuotedString
   alias.pop();
 }
     :
-           '"'
+           DoubleQuote
            ( escaped=ESC {lBuf.append(getText());} |
-             normal=~('"'|'\\'|'\n'|'\r')     {lBuf.appendCodePoint(normal);} )*
-           '"'
-           {setText(lBuf.toString());}
+             normal=~('"'|'“'|'”'|'\\'|'\n'|'\r')     {lBuf.appendCodePoint(normal);} )*
+           ( DoubleQuote {setText(lBuf.toString());} 
+           | ( '\n' | '\r')  {wibble("Found newline in string "+lBuf.toString(), input, false);})
```

Comment:
Please don't use "wibble". When you get parse errors etc. we get "wibble" in the output and it just looks amateurish.